apps/dotcom/sync-worker/src/TLPostgresReplicator.ts

```diff
@@ -863,7 +863,9 @@ export class TLPostgresReplicator extends DurableObject<Environment> {
 			this.reportActiveUsers()
 			this.log.debug('inserted active user')
 
+			const resumeStart = Date.now()
 			const resume = this.getResumeType(lsn, userId, guestFileIds)
+			this.logEvent({ type: 'resume_duration', duration: Date.now() - resumeStart })
```

Comment:
ope, just remembered that this won't work when deployed remotely due to the security thing cloudflare do. dang.